tokio/tests/task_local_set.rs

```diff
@@ -573,6 +573,27 @@ async fn spawn_wakes_localset() {
     }
 }
 
+/// Checks that the task wakes up with `enter`.
+/// <https://github.com/tokio-rs/tokio/issues/5020>.
```

Comment:
nit, not important: I might add:
```suggestion
/// Reproduces <https://github.com/tokio-rs/tokio/issues/5020>.
```